graph/schemas/mutations.graphql

```diff
@@ -0,0 +1,24 @@
+type Mutation {
```

Comment:
You can separate mutation and queries to their respective files using `extend type Mutation` instead of putting all the mutations/queries in a single file.